### PR TITLE
feat(tld): added localhost to accepted TLDs

### PR DESCRIPTION
**Pull request by lake-effect**
This allows the common TLD for 127.0.0.1 to be used with mehserve as
well (see https://en.wikipedia.org/wiki/.localhost).

I'm still working on this but I wanted to propose the idea for discussion.

### CODE REVIEW

**Comment by benjie** [comment]
Sounds good to me. What OS are you on?

**Comment by lake-effect** [comment]
I'm on Ubuntu LTS, but I'll be testing this with workmates on OSX.

**Comment by benjie** [comment]
Fantastic. I was wondering if it'll need a resolver installed for OSX or not (like we have for the other two domains).

**Comment by remindmmeinzer** [comment]
Can you clarify what you mean by resolver?

I'm working on troubleshooting getting this change working on OSX

**Comment by remindmmeinzer** [comment]
Just realized what you meant. `sudo cp /etc/resolver/meh /etc/resolver/localhost` got it working as expected!

File: README.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ To set up a subdomain, simply run
 
 `mehserve add mysite 1337`
 
-This'll tell mehserve to proxy all HTTP requests for `mysite.meh`
+This'll tell mehserve to proxy all HTTP requests for `mysite.meh, mysite.localhost`,
```
Comment (benjie) [inline README.md]:
```suggestion
This'll tell mehserve to proxy all HTTP requests for `mysite.meh`, `mysite.localhost`,
```

File: src/ssl.js [hunk 2]
```diff
@@ -45,6 +45,7 @@ extendedKeyUsage = serverAuth
 subjectAltName = @alt_names
 [alt_names]
 DNS.1 = ${hostname}
+DNS.2 = ${hostname}.localhost
```
Comment (benjie) [inline src/ssl.js]: Please add to the end with `DNS.4 =`; we don't want conflicting numbers :+1:

**Review by benjie — CHANGES_REQUESTED** [review]
Looking good! You may need to rerun prettier after accepting the backtick tweak.

File: README.md [hunk 3]
```diff
@@ -87,7 +87,7 @@ To set up a subdomain, simply run
 
 `mehserve add mysite 1337`
 
-This'll tell mehserve to proxy all HTTP requests for `mysite.meh`
+This'll tell mehserve to proxy all HTTP requests for `mysite.meh`, mysite.localhost`,
```
Comment (benjie) [inline README.md]:
```suggestion
This'll tell mehserve to proxy all HTTP requests for `mysite.meh`, `mysite.localhost`,
```

File: README.md [hunk 4]
```diff
@@ -97,7 +97,7 @@ Alternatively, to serve static files:
 `mehserve add staticsite /path/to/public`
 
 This'll tell mehserve to serve static content from `/path/to/public/` to anyone
-requesting `http://staticsite.meh/`
+requesting `http://staticsite.meh/`and `http://mysite.localhost`
```
Comment (benjie) [inline README.md]:
```suggestion
requesting `http://staticsite.meh/` or `http://mysite.localhost`
```

**Review by benjie — CHANGES_REQUESTED** [review]

**Review by benjie — APPROVED** [review]
LGTM :+1:

**Comment by benjie** [comment]
2.1.0 released :tada:

**Pull request merged**